cmd/atlas/inspect.go

```diff
@@ -31,3 +36,16 @@ func init() {
 	)
 	cobra.CheckErr(inspectCmd.MarkFlagRequired("dsn"))
 }
+
+func runInspect(dsn string) {
+	ctx := context.Background()
+	a, err := defaultMux.OpenAtlas(dsn)
+	cobra.CheckErr(err)
```

Comment:
feels weird to me to use a library for such a simple thing why not 
```suggestion
	if a, err := defaultMux.OpenAtlas(dsn); err != nil {
	  log.Fatalf("Error: %s", err)
	}
```
Or similar?

Reply:
it's the cobra std way, and its less lines.